proto/logbackuppb.proto

```diff
@@ -34,10 +34,25 @@ message GetLastFlushTSOfRegionResponse {
     repeated RegionCheckpoint checkpoints = 1;
 }
 
+message SubscribeFlushEventRequest {
+    string client_id = 1;
```

Comment:
Is this `client_id` is "store_id" ?

Reply:
This can be a random string, just for debugging usage.